crates/l2/prover/src/guest_program/src/risc0/Cargo.toml

```diff
@@ -33,7 +33,7 @@ k256 = { git = "https://github.com/risc0/RustCrypto-elliptic-curves", tag = "k25
 ed25519-dalek = { git = "https://github.com/risc0/curve25519-dalek", tag = "curve25519-4.1.2-risczero.0" }
 rsa = { git = "https://github.com/risc0/RustCrypto-RSA", tag = "v0.9.6-risczero.0" }
 crypto-bigint = { git = "https://github.com/risc0/RustCrypto-crypto-bigint", tag = "v0.5.5-risczero.0" }
-c-kzg = { git = "https://github.com/risc0/c-kzg-4844", tag = "c-kzg/v2.1.1-risczero.0" }
+c-kzg = { git = "https://github.com/lambdaclass/risc0-c-kzg-4844.git", tag = "c-kzg/v2.1.1-risczero.2-eip7594-default" }
```

Comment:
👍

Reply:
Let's add a descriptive comment explaining why we are using a fork for this.

Reply:
No longer needed after fix [484cb7c](https://github.com/lambdaclass/ethrex/pull/4712/commits/484cb7cbb64d69441456d58c0fd39e687b419d7a)